app/assets/javascripts/components/articles/article_list.jsx

```diff
@@ -215,14 +215,20 @@ const ArticleList = createReactClass({
       </div>
     );
 
-    let showMoreButton;
+    let showMore;
     if (!this.props.limitReached) {
-      showMoreButton = <div><button className="button ghost stacked right" onClick={this.showMore}>{I18n.t('articles.see_more')}</button></div>;
+      showMore = (
```

Comment:
I suggest a more explicit name like `showMoreSection` so that this won't get confused with the function (this.seeMore) that has the same name.